src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -86,6 +89,22 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         }
     }
 
+    shouldHandleWidgetDataChange() {
+        return this.isPaymentCharged();
+    }
+
+    isPaymentCharged() {
+        if (isEmptyObject(this.PAYMENT_TRANSACTION_STATUS_ENUM)) {
```

Comment:
Should this condition return true?  I would think blank transaction status would indicate that the payment is NOT charged, so maybe it should move into the "shouldHandleWidgetDataChange()" function?

Reply:
(or is this what we talked about returning false during dev huddle today?

Reply:
@ptewson-sfdo Just pushed an update to make this logic more clear. Since we get the payment status enum object in the connectedCallback(), the `widgetDataFromState` setter will execute before the connectedCallback() completes so we don't want to either allow the widget data change to be handled when the status enum object is empty for the initial mounting of the widget or allow form state changes to be handled when we have any other status than 'CAPTURED' and 'SUBMITTED'. So I moved the empty object check for `this.PAYMENT_TRANSACTION_STATUS_ENUM` to the setter itself.